src/config/selectors.js

```diff
@@ -18,3 +18,6 @@ export const GRAASP_SELECTION_TITLE_ID = 'graaspSelectionTitleId';
 export const DISCOVER_SECTION_TITLE_ID = 'discoverSectionTitleId';
 
 export const buildMyListNavigationTabId = (index) => `nav-tab-${index}`;
+
+export const MY_FAVORITES_COLLECTIONS_ID = 'favoriteCollectionsGrid';
+export const MY_LIKES_COLLECTIONS_ID = 'likedCollectionsGrid';
```

Comment:
It might be my mistake, but we shouldn't have multiple `selectors.js` file. Let's keep only the `src/config/selectors.js` one.